pages.ne/common/cat.md

```diff
@@ -0,0 +1,24 @@
+# cat
+
+> फाइलहरू देखाउनुहोस् र जोड्नुहोस्।
+> थप जानकारी: <https://www.gnu.org/software/coreutils/cat>
+
+- फाइल भित्रका कुराहरुलाई मानक आउटपुटमा देखाउनुहोस्:
+
+`cat {{फाइलको/बाटो}}`
+
+- धेरै फाइलहरू जोडेर एउटा सिंगो आउटपुट फाइल बनाउनुहोस्:
+
+`cat {{पहिलो/फाइलको/बाटो}} {{दोस्रो/फाइलको/बाटो}} > {{आउटपुट/फाइलको/बाटो}}`
+
+- धेरै फाइलहरू जोडेर एउटा सिंगो आउटपुट फाइलमा संलग्न गर्नुहोस्:
+
+`cat {{पहिलो/फाइलको/बाटो}} {{दोस्रो/फाइलको/बाटो}} >> {{आउटपुट/फाइलको/बाटो}}`
```

Comment:
```suggestion
`cat {{फाइल_पहिलो/को/पथ}} {{फाइल_दोस्रो/को/पथ}} >> {{आउटपुट/फाइल/को/पथ}}`
```
This can be used or
```suggestion
`cat {{फाइल1/को/पथ}} {{फाइल2/को/पथ}} >> {{आउटपुट/फाइल/को/पथ}}`
```